extension/package.json

```diff
@@ -5,11 +5,13 @@
   "license": "MIT",
   "scripts": {
     "check-coverage": "nyc check-coverage --statements 97",
-    "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
+    "test": "npm run check && npm run cover:unit && npm run cover:integration-plus-e2e && npm run check-coverage",
```

Comment:
I don't understand this change. Why you need to make it like this? I thought we don't clean up any resources anymore?

Reply:
Keep payments and types all the time when test server is started.
https://github.com/commercetools/commercetools-adyen-integration/pull/837/commits/1c9c2e4b6290b8ea7b3c49ce0d0fdb9f9cf5ce81